### ISSUE TITLE
Abstract `Invoices` from the UI and use `Payment Request` when user wants to create an invoice.

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver#3780]
Currently, the distinction between payment request and invoices is not really clear. The only difference is that payment request provides user with a rate at the exact moment and generates invoice for it, while invoices have expiration time.

For users, one of the first things they do when they deploy btcpay after the initial setup is go to the `Invoices` create an invoice. But this isn't the invoice they were expecting to create and it really confuses people, as now they only have 15 minutes to get paid, while traditionally they were expecting to invoice a client.

Invoices are just a checkout page, that user doesn't have to interact with in the UI, it's the customers who interact with it.

This issue is opened for a wider discussion on how to unify invoices and payment requests in the UI.

**Comment by dennisreimann** [comment]
I agree and think it'll be a good improvement to abstract this distinction away. Most users expect the functionality of payment requests from what they think of as invoices — and if they want a fixed BTC/SATS amount that can be achieved with payment requests as well.

In the API we should definitely have and keep that distinction — there we are dealing with more technically focused users that might care about the difference.

**Comment by dstrukt** [comment]
Conceptual ACK.

Have a lot to say here, will update my message.

**Comment by dennisreimann** [comment]
Let's address this in the next dev meeting and talk about how we can proceed.
